deploy/charts/monitoring/templates/grafana.yaml

```diff
@@ -70,3 +70,6 @@ data:
 {{ .Files.Get "dashboards/cluster-health.json" | indent 4 }}
   dataplane.json: |-
 {{ .Files.Get "dashboards/dataplane.json" | indent 4 }}
+  node-scanning.json: |-
+{{ .Files.Get "dashboards/node-scanning.json" | indent 4 }}
+
```

Comment:
I thought we are not committing the Grafana dashboard 🤔

Reply:
Anyway, I looked at the dashboards and they are fine to be added. Of course they could be more fancy but maybe in a separate PR.

Reply:
The 30 minutes I understand, I was thinking that it will not be committed.

I think you should still consider not committing it, as when the FF is disabled, we will see an empty dashboard that may look broken to people who would be looking at it. We may keep the code and add the dashboard shortly before enabling the feature flag.

Reply:
That's a very good point. I've removed them from the PR and set up a new ticket (ROX-14626) to add them back before we pull the FF

Reply:
> Before you merge, I would kindly ask you to deploy it on the test cluster with FF enabled **and disabled** to check whether the nil-checks are working correctly (especially when inventory==nil).

I have tested the latest version by switching the feature flag to off in the test cluster, followed by a clean teardown and redeploy with disabled feature flag.
All looks stable, I did not observe any crashes,  neither of sensor nor of compliance.